dms/models/storage.py

```diff
@@ -85,6 +85,19 @@ class Storage(models.Model):
     )
 
     model_ids = fields.Many2many("ir.model", string="Linked Models")
+    inherit_access_from_parent_record = fields.Boolean(
+        string="Inherit permissions from related record",
+        default=False,
```

Comment:
IMHO this is the most sane default (see TT29500).

```suggestion
        default=True,
```

Reply:
No, this shouldn't be enabled by default, as it's an opt-in feature, and even more, you will be changing current behavior on existing installations.

Reply:
Setting `False` here means that you want to bypass Odoo's security model. It opens a door for data leaks, so IMHO it should be `True` by default.

The behavioural change is true, though. That can be fixed with a migration script.

Reply:
As said, this should continue as it was before, with no change. If you want it, then you enable it.

Reply:
Could you please elaborate why?

Reply:
I have already told you. It's this way also in 12.0. And even more: this only makes sense if the storage is attachment. I would accept an onchange if you change the storage type to attachment.

Reply:
Ah OK, @victoralmau could you please add the onchange?